libs/community/tests/unit_tests/llms/test_predibase.py

```diff
@@ -17,3 +17,20 @@ def test_api_key_masked_when_passed_via_constructor(
     captured = capsys.readouterr()
 
     assert captured.out == "**********"
+
+
+def test_specifying_adapter_id_argument() -> None:
```

Comment:
```suggestion
@pytest.mark.requires("predibase")
def test_specifying_adapter_id_argument() -> None:
```

Reply:
@efriis I am happy to add this marker, but it is not required, because the `Predibase` library itself is not required to run these tests (they only instantiate `LangChain` side objects).  Should I still add it?  Please let me know.  Thank you.